content/odin/foundations/html_css/flexbox/project-landing-page.md

```diff
@@ -0,0 +1,70 @@
+### Giriş
+
+Bu proje için size sağlayacağımız bir tasarımdan tüm bir web sayfası oluşturacaksınız. Eğer takip ettiyseniz, bunu başarmak için gereken becerilere sahip olmalısınız, ancak kolay olmayabilir!
+
+Size sunduğumuz tasarım 2 resimden oluşuyor: biri web sitesinin tamamının resmi, diğeri ise kullandığımız bazı yazı tipleri ve renkler hakkında bazı ayrıntılar içeriyor.
+
+Bir şeye bakmak için google kullanmaktan veya önceki derslere geri dönmekten korkmayın. **Gerçek hayatta, profesyonel geliştiriciler yıllardır yaptıkları şeyler için google'ı sürekli kullanırlar.** Bu noktada her şeyi ezberlemiş olmanız beklenmemektedir, bu nedenle bu konuda endişelenmeyin. Ayrıca, derslerimizde henüz karşılaşmamış olabileceğiniz birkaç küçük ayrıntı var. Bu ayrıntılar önemsizdir ve kolayca aranabilir. (örneğin google `css rounded corners`).
+
+Projenizi tasarıma olabildiğince benzetmeye çalışın, ancak piksel mükemmelliğine ulaşma konusunda endişelenmeyin. Çeşitli bölümler arasındaki tam kenar boşluklarını bulmak için cetvelinizi çıkarmayın veya pikselleri saymayın. Bu ödevin amacı, sıfırdan bir şey yaratmak ve çeşitli unsurları diğerlerine göre aşağı yukarı doğru konuma getirmektir. Tasarımın gerçekte `margin`i varken `margin: 24px` kullanmanız ya da `48px` olması önemli değildir.
+
+Son olarak_, bu tasarıma kendi içeriğinizi eklemekten çekinmeyin. Resimlerde bazı anlamsız kukla içerikler var, ancak bir iş uydurmak ve bu sayfayı kişiselleştirmek istiyorsanız, lütfen bunu yapmaktan çekinmeyin! Yer tutuculara gerçek resimler ekleyin ve renklerle ve yazı tipleriyle de biraz oynamaktan çekinmeyin.

```

Comment:
```suggestion
_Son olarak_, bu tasarıma kendinizden eklemeler yapmaktan çekinmeyin. Resimlerde bazı anlamsız içerikler var, ancak biraz zaman harcayıp bu sayfayı kişiselleştirmek istiyorsanız, lütfen bunu yapmaktan çekinmeyin! Yer tutuculara gerçek resimler ekleyin, renkleri ve yazı tiplerini de kurcalamaktan çekinmeyin.

```
original sentence;

"The images have some meaningless dummy content, but if you want to make up a business and personalize this page, please feel free to do so!"

make up a business bölümü burada biraz daha mesai/zaman harcamak anlamında kullanılmış